tests/integration/targets/nd_compliance_analysis/tasks/main.yml

```diff
@@ -0,0 +1,131 @@
+# Test code for the ND modules
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Add check for the used variables like site_name and insights_group

Reply:
ok